crates/admin/src/cluster_controller/protobuf.rs

```diff
@@ -21,11 +18,11 @@ pub const FILE_DESCRIPTOR_SET: &[u8] =
 /// Creates a new ClusterCtrlSvcClient with appropriate configuration
 #[cfg(feature = "clients")]
 pub fn new_cluster_ctrl_client(
-    channel: Channel,
-) -> cluster_ctrl_svc_client::ClusterCtrlSvcClient<Channel> {
+    channel: tonic::transport::Channel,
```

Comment:
I am just wondering why this was necessary ?

Reply:
The first commit is unrelated to the hardening. It's to avoid compiler warnings of unused imports when not activating the "clients" feature.